.changeset/rare-toes-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-devops-backend': patch
```

Comment:
To @benjdlambert point, this would be a breaking change right? so `minor`

Reply:
Yes, but I'd like to refactor this, I think there is a non-break approach that I'd like to take that will also improve a few things